src/store/index.js

```diff
@@ -24,12 +24,14 @@ const reducer = combineReducers({
   routing,
   browser: responsiveStateReducer,
 });
+/* eslint-disable no-underscore-dangle */
```

Comment:
Could we just disable globally?  I'd prefer not to have eslint disables sprinkled around especially if they're permanent